src/internal_modules/roc_sndio/pump.cpp

```diff
@@ -81,6 +83,26 @@ void Pump::stop() {
     stop_ = 1;
 }
 
+status::StatusCode Pump::close_device_(IDevice* device) {
```

Comment:
nit: Please follow the same method order in .cpp file as they're declared in .h file. Same for other classes.